### PR TITLE
fix(sct_runner): added tags to sct runners

### PR DESCRIPTION
**Pull request by rayakurl**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/sct_runner.py [hunk 1]
```diff
@@ -381,15 +381,16 @@ def create_instance(self, test_id: str, test_duration: int,  # pylint: disable=t
                          self.region_name, self.CLOUD_PROVIDER, self.region_name)
             return None
 
-        tags = {
+        tags = TestConfig.common_tags()
+        tags.update({
             "TestId": test_id,
+            "TestName": test_name,
             "NodeType": self.NODE_TYPE,
-            "RunByUser": get_username(),
```
Comment (fgelcer) [inline sdcm/sct_runner.py:387]: why are you removing this?

**Review by fgelcer — CHANGES_REQUESTED** [review]
everything looks fine, but removing `RunByUser` label IMHO

File: sdcm/sct_runner.py (hunk 1 above)
Comment (rayakurl) [reply sdcm/sct_runner.py:387]: Because it's already exists in TestConfig.common_tags

**Comment by rayakurl** [comment]
Screenshot from 2023-01-24 17-09-37
Screenshot from 2023-01-24 17-08-50
unnamed

File: sdcm/sct_runner.py (hunk 1 above)
Comment (rayakurl) [reply sdcm/sct_runner.py:387]: correct. But at this point, its value is None

Comment (fgelcer) [reply sdcm/sct_runner.py:387]: this is the kind of information that should be in the commit message + commit description

Comment (rayakurl) [reply sdcm/sct_runner.py:387]: @fgelcer - done

**Comment by roydahan** [comment]
@rayakurl please check the failed checks.
Need to verify this reaches also sct_plugin and run in all siren-tests jobs.

File: sdcm/sct_runner.py [hunk 2]
```diff
@@ -52,7 +52,7 @@
 from sdcm.utils.azure_region import AzureOsState, AzureRegion, region_name_to_location
 from sdcm.utils.get_username import get_username
```
Comment (vponomaryov) [inline sdcm/sct_runner.py:53]: Linter fails because this import is unused. Need to delete it.

**Review by fruch — APPROVED** [review]
LGTM,

Just fix the unused import

**Review by ShlomiBalalis — DISMISSED** [review]
Marvelous! Just please remove the unnecessary import

File: sdcm/sct_runner.py (hunk 2 above)
Comment (rayakurl) [reply sdcm/sct_runner.py]: @vponomaryov - done

**Comment by rayakurl** [comment]
> @rayakurl please check the failed checks. Need to verify this reaches also sct_plugin and run in all siren-tests jobs.

Will do

**Review by fruch — APPROVED** [review]

**Comment by roydahan** [comment]
Who is doing the backports?

**Comment by fgelcer** [comment]
> Who is doing the backports?

@roydahan , thank you for volunteering

**Comment by fruch** [comment]
> > Who is doing the backports?
> 
> @roydahan , thank you for volunteering

I would recommend also backporting 
https://github.com/scylladb/scylla-cluster-tests/pull/5730

Since it fixes a bug introduced by this PR

**Comment by fgelcer** [comment]
> I would recommend also backporting #5730
> 
> Since it fixes a bug introduced by this PR

technically speaking, for 5.2 and 2023.1 it should be already in there (excluding this fix above)
@fruch , can you do the backports, as i think Roy won't be available for that soon

**Comment by rayakurl** [comment]
@fruch , @enaydanov - can you please backport?

**Comment by enaydanov** [comment]
@rayakurl please prepare a separate PR for 2022.1

**Pull request merged**